<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormExaminee</class>
 <widget class="QWidget" name="FormExaminee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,0,0,0,1,2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Звание:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_rang"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Войсковая часть:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_department"/>
     </item>
     <item>
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Подразделение:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_subdivision"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>37</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Фамилия:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_surname">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Имя:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_name">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Отчество:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_middleName">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Дата рождения:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_born"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Возрастная группа:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_ageGroup"/>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Группа:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_group"/>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Подгруппа:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_subGroup"/>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Вес:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="dsb_weght">
       <property name="minimum">
        <number>80</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Диагноз:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_diagnos"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Сопутствующие заболевания:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="text_diseases">
       <property name="plainText">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="2,3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Примечание:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="text_comments">
         <property name="plainText">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Результаты тестирования:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QTableWidget" name="tableWidget">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Дата</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Кол-во упр.</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Сумма балов</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Пороговый балл</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QToolButton" name="btn_addTest">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../res.qrc">
               <normaloff>:/images/images/add.png</normaloff>:/images/images/add.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btn_deleteTest">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../res.qrc">
               <normaloff>:/images/images/delete.png</normaloff>:/images/images/delete.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btn_editTest">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../res.qrc">
               <normaloff>:/images/images/edit.png</normaloff>:/images/images/edit.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
